django/contrib/admindocs/views.py

```diff
@@ -404,8 +404,15 @@ def get_context_data(self, **kwargs):
             # Non-trivial TEMPLATES settings aren't supported (#24125).
             pass
         else:
-            # This doesn't account for template loaders (#24128).
-            for index, directory in enumerate(default_engine.dirs):
+            directories = list(default_engine.dirs)
+            for loader in default_engine.template_loaders:
+                directories.extend(loader.get_dirs())
```

Comment:
_Assuming_ (🤷) my other remark about `get_dirs` is correct, I'd presume this line needs a `if hasattr(loader, "get_dirs")` preceding it...

Reply:
Yes, I think you are right. 👍🏼

Thanks, that you took the time to have a look at this PR! 🙏🏼